src/fdk_rdf_parser/parse_functions/concept.py

```diff
@@ -318,6 +318,7 @@ def parse_concept(graph: Graph, fdk_record_uri: URIRef, concept_uri: URIRef) ->
         partitiveRelation=extract_partitive_relations(graph, concept_uri),
         genericRelation=extract_generic_relations(graph, concept_uri),
         created=date_value(graph, concept_uri, DCTERMS.created),
+        creator=object_value(graph, concept_uri, DCTERMS.creator),
```

Comment:
Vi vil være interessert i mer enn bare uri'en til creator, du kan bruke samme metode som brukes for pusblisher. Men da må vi legge til tripler for creator ila jobben til reasoning-service

Reply:
OK. Kan ta `creator` is sin egen pr :)